### DISCUSSION TITLE
Enable use of grafana environment variables in dashboards.

### DISCUSSION THREAD

**Discussion by zachmailloux** [grafana/grafana#38397] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
The ability to use grafana environment variables in my individual dashboards

**Why is this needed**:
Throughout my multiple grafana environments I link out to various other sources, and would like to be able to use custom environment variables in my dashboards as variables. For instance, in my dev environment I could have an environment variable called "splunk_index_prefix" where all my hyperlinks sending to splunk can load that variable when displayed. This would allow me to change one value to enable the easy provisioning between environments.

**Comment by sakjur** [comment]
Related Slack thread: https://grafana.slack.com/archives/C05675Y4F/p1594142443499800

Practically we'll need to make sure to only ingest a safe set of environment variables (say prefixed by `GF_VAR_` or something like that) to avoid people extracting database access tokens and similar from the environment with this feature.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #83003. Feel free to continue the discussion around this there. Thank you!
